src/clang_parser.cpp

```diff
@@ -720,6 +720,7 @@ bool ClangParser::parse(ast::Program *program, BPFtrace &bpftrace, std::vector<s
     "-isystem", "/usr/local/include",
     "-isystem", "/bpftrace/include",
     "-isystem", "/usr/include",
+    "-isystem", "/usr/include/x86_64-linux-gnu",
```

Comment:
Would this be an issue on !x86 systems w/ x86 headers installed? Should we get host arch and craft an arch-specific path?

Reply:
Right, yeah, that could be an issue. I'll try to craft the path from the host arch.